alembic/env.py

```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
+from sqlalchemy import engine_from_config, pool
+from alembic import context
+from database import Base
+from city.models import City
```

Comment:
Verify that the `City` model is correctly defined and imported from `city.models`. This is necessary for Alembic to recognize changes in the model for migrations.